app/controllers/supervisor/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Supervisor::CommentsController < Supervisor::BaseController
+
+  def create
+    comment = Comment.new(comment_params)
+
+    if comment.save
```

Comment:
@andremedeiros @nathanvda Haha, now I have to find a way that satisfies both of you. Stakes are up.
Nathan: We didn't validate text presence, because the supervisor should be able to 'check' without text. Checks _without_ text should not be saved as a Comment; they only update the team's attribute `last_checked_at`. 
In both cases - check with or without text - the `:set_checked` method in the model is fired. How about I put the `if text ....`-clause within this method? Would this be unSOLID?